testsetups/ocr.go

```diff
@@ -230,6 +238,67 @@ func (t *OCRSoakTest) checkLatestRound(expectedValue, roundNumber int) {
 	}
 }
 
+// subscribeToAnswerUpdatedEvent subscribes to the event log for AnswerUpdated event and
+// verifies if the answer is matching with the expected value
+func (t *OCRSoakTest) subscribeToAnswerUpdatedEvent(ctx context.Context) {
+	contractABI, err := ethereum.OffchainAggregatorMetaData.GetAbi()
+	Expect(err).ShouldNot(HaveOccurred(), "Getting contract abi for OCR shouldn't fail")
+	query := goeath.FilterQuery{
+		Addresses: []common.Address{},
+	}
+	for i := 0; i < len(t.ocrInstances); i++ {
+		query.Addresses = append(query.Addresses, common.HexToAddress(t.ocrInstances[i].Address()))
+	}
+	eventLogs := make(chan types.Log)
+	sub, err := t.chainClient.SubscribeFilterLogs(context.Background(), query, eventLogs)
+	if err != nil {
+		Expect(err).ShouldNot(HaveOccurred(), "Subscribing to contract event log in OCR instance shouldn't fail")
+	}
+	ocr := t.ocrInstances[0]
+	go func() {
+		//defer GinkgoRecover()
```

Comment:
Do we need this? I know GinkgoRecover can be really wonky in these goroutines sometimes so I am fine with leaving the comment here.